api/pendingactions/actions/actions.go

```diff
@@ -1,7 +1,7 @@
 package actions
 
 const (
-	CleanNAPWithOverridePolicies = "CleanNAPWithOverridePolicies"
-	DeleteK8sRegistrySecrets     = "DeleteK8sRegistrySecrets"
-	PostInitMigrateEnvironment   = "PostInitMigrateEnvironment"
+	CleanNAPWithOverridePolicies      = "CleanNAPWithOverridePolicies"
+	DeletePortainerK8sRegistrySecrets = "DeletePortainerK8sRegistrySecrets"
```

Comment:
This should not have been renamed originally. So I'm moving it back. Just in case there is a pending action with this name preexisting.